### ISSUE TITLE
0 byte parquet file crashes pyarrow and the cli

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#57]
I don't know how, but somehow I got a 0 length parquet file
```
(suzieq) jpiet@t14:/tmp/pycharm_project_304/suzieq$ ls -l ~/parquet-out/ifCounters/namespace\=dual-ospf/hostname\=internet/9c3e9450893449d3ada181d78be808a6.parquet
-rw-rw-r-- 1 jpiet jpiet 0 Mar 30 16:46 '/home/jpiet/parquet-out/ifCounters/namespace=dual-ospf/hostname=internet/9c3e9450893449d3ada181d78be808a6.parquet'
```
it produces this output in the cli when table show
```
jpiet> table show                                                                                                            Error: 1Error running command: Invalid: Parquet file size is 0 bytes
------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/nubia/internal/cmdbase.py", line 374, in run_interactive
    ret = fn(**args_dict)
  File "/tmp/pycharm_project_304/suzieq/suzieq/cli/sqcmds/TableCmd.py", line 40, in show
    df = self.sqobj.get(hostname=self.hostname, namespace=self.namespace)
  File "/tmp/pycharm_project_304/suzieq/suzieq/sqobjects/tables.py", line 25, in get
    info.update(table_obj.get_table_info(table, **kwargs))
  File "/tmp/pycharm_project_304/suzieq/suzieq/engines/pandas/engineobj.py", line 178, in get_table_info
    all_time_df = self._get_table_info(table, view='all', **kwargs)
  File "/tmp/pycharm_project_304/suzieq/suzieq/engines/pandas/engineobj.py", line 204, in _get_table_info
    **kwargs
  File "/tmp/pycharm_project_304/suzieq/suzieq/engines/pandas/engine.py", line 158, in get_table_df
    .read(columns=fields)
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pyarrow/parquet.py", line 1137, in read
    use_pandas_metadata=use_pandas_metadata)
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pyarrow/parquet.py", line 591, in read
    reader = self.open()
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pyarrow/parquet.py", line 567, in open
    reader = self.open_file_func(self.path)
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pyarrow/parquet.py", line 942, in _open_dataset_file
    buffer_size=dataset.buffer_size
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pyarrow/parquet.py", line 137, in __init__
    read_dictionary=read_dictionary, metadata=metadata)
  File "pyarrow/_parquet.pyx", line 1036, in pyarrow._parquet.ParquetReader.open
  File "pyarrow/error.pxi", line 99, in pyarrow.lib.check_status
OSError: Invalid: Parquet file size is 0 bytes
------------------------------------------------------------
```

**Comment by jopietsch** [comment]
I have another 0 byte file and table show doesn't fail, but it a specific table doesn't show.

In other words, I have a 0 byte file in ifCounters

```
(suzieq-QZqWZG2D-py3.8) jpietsch@jpietsch-server:~/suzieq$ suzieq/cli/suzieq-cli table show
         table                       first_time                      latest_time  intervals  all rows  namespaces  devices
0        arpnd 2020-08-15 21:59:06.424000-07:00 2021-03-07 16:42:31.224000-08:00        237      4592          10       27
1          bgp 2020-08-16 00:53:20.544000-07:00 2021-03-07 16:42:32.982000-08:00        190      1441           9       15
2       device 2020-08-15 21:59:05.877000-07:00 2021-03-07 16:42:30.874000-08:00        226       575          10       27
3      evpnVni 2020-11-21 19:56:34.484000-08:00 2021-03-07 16:42:33.391000-08:00        104       458           7        9
4           fs 2020-08-15 21:59:06.440000-07:00 2021-03-08 00:35:08.685000-08:00        589     14368          10       27
5   interfaces 2020-08-15 21:59:05.684000-07:00 2021-03-07 16:42:30.874000-08:00        237      6158          10       27
6         lldp 2020-08-15 21:59:05.877000-07:00 2021-03-07 16:42:31.547000-08:00        158      1353          10       14
7         macs 2020-08-15 21:59:05.222000-07:00 2021-03-07 16:42:32.259000-08:00        168      3530          10       24
8         mlag 2020-08-16 00:53:20.845000-07:00 2021-03-07 16:42:31.547000-08:00         64       115           6        5
9       ospfIf 2020-08-15 21:59:06.603000-07:00 2021-02-10 17:46:18.806000-08:00         88       556           9       12
10     ospfNbr 2020-08-15 21:59:06.548000-07:00 2021-02-10 17:46:16.679000-08:00        191       611           9       12
11      routes 2020-08-15 21:59:05.270000-07:00 2021-03-07 16:42:32.982000-08:00        282      8593          10       27
12    sqPoller 2021-02-04 16:29:53.679000-08:00 2021-03-08 11:23:04.432000-08:00    1175945   1265830           9       54
13        time 2020-08-15 21:59:05.270000-07:00 2021-03-07 16:42:31.824000-08:00        239       567          10       25
14      topcpu 2020-08-15 21:59:06.482000-07:00 2021-03-08 11:21:40.486000-08:00     418634   2540268          10       25
15      topmem 2020-08-15 21:59:06.486000-07:00 2021-03-08 11:22:43.268000-08:00     381028   3662762          10       25
16        vlan 2020-08-15 21:59:05.879000-07:00 2021-03-07 16:42:32.471000-08:00        111       590          10       12
17       TOTAL 2020-08-15 21:59:05.222000-07:00 2021-03-08 11:23:04.432000-08:00    1175945   7512367          10       54

```

**Comment by ddutt** [comment]
This has been fixed with the commit 41c1001600b49c9ecaf3d1a9b594dd5968173316, 4fa3a770814d9a983ff100c7a62ab7c5169180b1
